examples/complex_query.rs

```diff
@@ -66,18 +64,18 @@ fn main() {
     let expected = "
    SELECT *
      FROM bazaar.product
-          LEFT JOIN bazaar.product_category 
-          ON product_category.product_id = product.product_id 
-          LEFT JOIN bazaar.category 
-          ON category.category_id = product_category.category_id 
-          LEFT JOIN product_photo 
-          ON product.product_id = product_photo.product_id 
-          LEFT JOIN bazaar.photo 
-          ON product_photo.photo_id = photo.photo_id 
-    WHERE product.name = $1 
-      AND category.name = $2 
- GROUP BY category.name 
-   HAVING count(*) > $3 
+          LEFT JOIN bazaar.product_category\x20
+          ON product_category.product_id = product.product_id\x20
+          LEFT JOIN bazaar.category\x20
+          ON category.category_id = product_category.category_id\x20
+          LEFT JOIN product_photo\x20
+          ON product.product_id = product_photo.product_id\x20
+          LEFT JOIN bazaar.photo\x20
+          ON product_photo.photo_id = photo.photo_id\x20
+    WHERE product.name = $1\x20
+      AND category.name = $2\x20
+ GROUP BY category.name\x20
+   HAVING count(*) > $3\x20
  ORDER BY product.name ASC, product.created DESC".to_string();
```

Comment:
There were 3 issues with the previous version:
1. If you don't have an editor that highlights whitespace at the end of lines, you can't really tell that there were whitespace in the first place.
2. A failing test shows the difference between the result and the expected output but you can't see what's missing. 
3. Many editors (including mine) are configured to strip whitespace at the end of lines which makes it difficult to edit without breaking the tests.